plugin/src/main/scala/chisel3/internal/plugin/BundleComponent.scala

```diff
@@ -189,6 +194,49 @@ private[plugin] class BundleComponent(val global: Global, arguments: ChiselPlugi
       elementsImpl
     }
 
+    def generateAutoTypename(bundle: ClassDef, thiz: global.This): Option[Tree] = {
+      // 1: Get immediate defined args of the Bundle
+      // 2: Determine an appropriate naming strategy based on the collection of elements:
+      //    - 1 numeric property only: Generate a name like MyBundle2
```

Comment:
```suggestion
      //    - 1 numeric property only: Generate a name like MyBundle2
```

don't the test examples show this is still like `MyBundle_2`?

Reply:
Sorry, the comments are outdated, I need to update them